src/Cassandra.IntegrationTests/Core/HeartbeatTests.cs

```diff
@@ -45,8 +45,8 @@ public void OneTimeTearDown()
             _testCluster.Remove().Wait();
         }
 
-        [TestCase(true)]
-        [TestCase(false)]
+        [TestCase(true), Category("short")]
```

Comment:
We can add the category to the fixture instead of a single test.

Reply:
done